packages/tools/pluggable-widgets-tools/jest.config.js

```diff
@@ -4,13 +4,13 @@ const nativeBaseConfig = require("./test-config/jest.native.config.js");
 const webConfig = {
     ...webBaseConfig,
     rootDir: ".",
-    testMatch: ["<rootDir>/src/{web,typings-generator}/**/*.spec.{ts,tsx}"]
+    testMatch: ["<rootDir>/src/@(web|typings-generator)/**/*.spec.ts"]
```

Comment:
You can theoretically write tests in `tsx`

Reply:
test marcher did not pick the 2 sources, needed to remove braces to make it work. As the current test are only `ts` files i suggest to keep the suggested matcher without tsx